src/Microsoft.Extensions.DependencyModel/project.json

```diff
@@ -14,7 +14,7 @@
   },
   "frameworks": {
     "net451": { },
-    "dotnet5.4": {
+    "dnxcore50": {
```

Comment:
Why was this change necessary? We need to use this library in Mvc, which targets `dotnet5.4`, not `dnxcore50`. Also, we should start moving away from `dnxcore50` anyway...